sssom/util.py

```diff
@@ -440,13 +461,19 @@ def dataframe_to_ptable(df: pd.DataFrame, priors=None, inverse_factor: float = 0
 RDF_FORMATS = {"ttl", "turtle", "nt", "xml"}
 
 
-def sha256sum(filename):
+def sha256sum(filename: str) -> str:
+    """Hashing function.
+
+    :param filename: Filename
```

Comment:
Would be a good time to rename this variable- `filename` is a bad variable. `path` would be better.